Application/src/main/res/values-de/strings.xml

```diff
@@ -175,12 +175,36 @@
     <string name="pref_key_disable_touchscreen_title">Touchscreen der Uhr deaktivieren</string>
 
     <!-- Summary for setting allowing the user to disable the watch touchscreen to avoid accidental touches -->
-    <string name="pref_key_disable_touchscreen_summary">Ist der Touchscreen der Uhr deaktiviert, muss man bei Wiederaktivierung, als erstes die Option deaktivieren und die Uhr erneut starten.</string>
+    <string name="pref_key_disable_touchscreen_summary">Ist der Touchscreen der Uhr deaktiviert, muss man bei Wiederaktivierung, als erstes die Option deaktivieren und die Uhr erneut starten</string>
 
     <!-- Title for setting to enable root -->
     <string name="pref_key_root_title">ROOT aktivieren</string>
 
-    <!-- Title for setting to enable root -->
+    <!-- Summary for setting to enable root -->
     <string name="pref_key_root_summary">Erfordert eine gerootete Uhr</string>
+    
+    <!--Title for deinstalling xDrip watchfece -->
+    <string name="pref_key_unistall_xDrip_watchface_title">Deinstalliere xDrip Ziffernblatt</string>
+    
+    <!-- Summary for removing xDrip watchfece from collector watch -->
+    <string name="pref_key_remove_xDrip_watchface_from_collector_watch_summary">Entferne das xDrip Ziffernblatt von der Sammleruhr</string> 
```

Comment:
Please note that this string name is incorrect against the current code, see: https://github.com/jamorham/LibreAlarm/blob/master/Application/src/main/res/values/strings.xml#L185